test/test_asynctcp.py

```diff
@@ -0,0 +1,222 @@
+import errno
```

Comment:
It looks like the current CI config runs `pytest || true`, presumably to skip the fact there aren't any tests. Might want to change that to plain `pytest`.